README.md

```diff
@@ -47,6 +45,57 @@ from the list provided above for compatibility purposes.
  * AMD Merlin/Falcon Embedded System is not currently supported by the public repo.
  * AMD Raven Ridge APU are currently not supported
 
+### New features and enhancements in ROCm 1.9.0
+
+#### Preview for Vega 7nm
+* Enables developer preview support for Vega 7nm
+
+#### System Management Interface
+* Adds support for the ROCm SMI (System Management Interface) library, which provides monitoring and management capabilities for AMD GPUs.
+
+#### Improvements to HIP/HCC
+* Support for gfx906
+* Added deprecation warning for C++AMP.  This will be the last version of HCC supporting C++AMP.
+* Improved optimization for global address space pointers passing into a GPU kernel
+* Fixed several race conditions in the HCC runtime
+* Performance tuning to the unpinned copy engine
+* Several codegen enhancement fixes in the compiler backend
+
+#### Preview for rocprof Profiling Tool
+Developer preview (alpha) of profiling tool 'rpl_run.sh', cmd-line front-end for rocProfiler, enables:
+* Cmd-line tool for dumping public per kernel perf-counters/metrics and kernel timestamps
+* Input file with counters list and kernels selecting parameters
+* Multiple counters groups and app runs supported
+* Output results in CSV format
+The tool location is: /opt/rocm/rocprofiler/bin/rpl_run.sh
+
+#### Preview for rocr Debug Agent rocr_debug_agent
+Developer preview (alpha) of Debug Agent
+* The rocr_debug_agent repository provides the components required to build the library for dumping GPU memory fault info.
+* The rocr_debug_agent components are used by ROCm-GDB to support debugging GPU kernels on ROCm. 
+* Enabled on Vega10 GPUs
+
+
+#### New distribution support 
+
+* Binary package support for Ubuntu 18.04
+
+#### ROCm 1.9 is ABI compatible with KFD in upstream Linux kernels. 
+Upstream Linux kernels support the following GPUs in these releases:
+4.17: Fiji, Polaris 10, Polaris 11
+4.18: Vega10
+
+Some ROCm features are not available in the upstream KFD:
+* Less system memory available to ROCm applications
```

Comment:
Our ROCm "feature" is that we offer less memory available? Or is this a problem that is *fixed* in ROCm compared to upstream?

Reply:
Right. This is inconsistent. I guess the missing feature is "More system memory available ..."